private/buf/cmd/buf/command/generate/generatev2.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020-2023 Buf Technologies, Inc.
```

Comment:
It doesn't make sense to have two separate entrypoints - you need to deduce within the single `run` command what version you have, and make inferences as to behavior based on that.